ciborium/src/value/canonical.rs

```diff
@@ -46,15 +46,14 @@ pub fn cmp_value(v1: &Value, v2: &Value) -> Ordering {
             Ordering::Equal => s.cmp(o),
             x => x,
         },
-        (Bool(s), Bool(o)) => s.cmp(o),
-        (Null, Null) => Ordering::Equal,
```

Comment:
Do these items need to be removed?

Reply:
Good catch, I don't think they do. I assume this an artifact of our first implementation, that removed `Value::Bool` and `Value::Null`, which we didn't properly undo.